src/userobjects/MyTRIMRasterizer.C

```diff
@@ -96,6 +96,7 @@ InputParameters validParams<MyTRIMRasterizer>()
   // Advanced options
   params.addParam<unsigned int>("interval", 1, "The time step interval at which TRIM BCMC is run");
   params.addParam<Real>("analytical_energy_cutoff", 0.0, "Energy cutoff in eV below which recoils are not followed explicitly but effects are calculated analytically.");
+  params.addParam<Real>("recoil_rate_scaling", 1.0, "A factor to scale computed reaction rates in the the PKAGenerator objects. This is useful to avoid extremely large PKA lists.");
```

Comment:
I don't quite understand what the difference between the scaling factor and a cap on total PKA number is. Wouldn't the latter automatically trigger result scaling?

Reply:
the number of pka you want to run requires assembling the pka list. The pka list can be really long (`1e12` and up) which takes forever to assemble and may overrun memory. Even if you discard most of them, there is still a bottleneck there.

Reply:
Ok, that makes sense. I still suggest we rename that other parameter.